ComparePaymentProcessors.md

```diff
@@ -0,0 +1,95 @@
+# Payment Processor Comparison
+
+Most new merchants will likely only consider the price of the service. Since BTCPay is free, that may have led you here and if so, welcome. 
+
+We want users to understand not only BTCPay, but also how payments can be processed on behalf of a merchant. In order to help users understand trade-offs when using various cryptocurrency payment processing methods. We don't know which payment processors provide which services. That will need to be part of your own research. The following list is a good place to start:
+
+* [Features](#Features)
+* [Security](#Security)
+* [Privacy](#Privacy)
+* [Decentralized](#Decentralized)
+* [Fiat](#Fiat)
+
+----
+
+## Features
+
+Every payment processor has features, here are some BTCPay features:
+
+* **Bitcoin** - Accepting Bitcoin is the first step.
+* **Altcoins** - Accept cryptocurrency alternatives to Bitcoin.
+* **Lightning** - Rapid Bitcoin microtransactions using the Lightning Network.
+* **Integrations** - Connect to your website using WooCommerce, Drupal, Magneto, Prestashop and custom integrations.
+* **Point Of Sale** - POS Interfaces for physical stores.
+* **Unlimited Stores** - Merchants can process payments for their own stores, or for others.
+* **Merchant Support** - 24/7 open source community support.
```

Comment:
I would remove this, we can't guarantee any sort of support, especially not 24/7, I don't want people to take that for granted.

Reply:
@bitcoinshirt it's just like how open source works. I'll remove it though. Otherwise everyone will be asking for you 24/7 :)

Reply:
well spotted, I shall also remove from btcpay jungle as well, like you said 
"I don't want people to take that for granted"  
Yes, Our Support at BTCPay Community is the best for sure.